htdocs/class/smarty/Config_File.class.php

```diff
@@ -145,7 +145,7 @@ function get($file_name, $section_name = NULL, $var_name = NULL)
     /**
      * Retrieves config info based on the key.
      *
-     * @param $file_name string config key (filename/section/var)
+     * @param string $file_name config key (filename/section/var)
```

Comment:
This directory should be a perfect copy of the upstream Smarty. No changes without very compelling and documented reasons.